package.json

```diff
@@ -36,6 +36,7 @@
     "eslint": "^2.10.2",
     "istanbul": "^0.4.3",
     "mocha": "^2.5.3",
+    "proxyquire": "^1.7.9",
```

Comment:
It makes it possible to stub dependencies. Do you have a better solution? Proxyquire is pretty standard for unit test